services/comm.cpp

```diff
@@ -1859,13 +1859,14 @@ void JobDoneMsg::send_to_channel(MsgChannel *c) const
     *c << flags;
 }
 
-LoginMsg::LoginMsg(unsigned int myport, const std::string &_nodename, const std::string _host_platform)
+LoginMsg::LoginMsg(unsigned int myport, const std::string &_nodename, const std::string &_netname, const std::string _host_platform)
```

Comment:
we had to add netname to login message so that we know it is OK to disconnect a client if their netname does not match with schedulers netname